### ISSUE TITLE
How to run the project

### ISSUE THREAD

**Issue by zaynaab** [brayton-anderson/preeclampsia-flutter-app#1]
Hi,
I'm a researcher and I'm doing research on early prediction of eclampsia; I want to collect data for research, and I found this app helpful but I've no idea how to run this code. I will be very thankful if you guide me in regard.

**Comment by brayton-anderson** [comment]
Hi,

Kindly give me a few days to update the read me file. Ill notify you here.

Kind regards,
Brayton Anderson.

Innovator, Embedded Systems Engineer & Full Stack Developer
I offer a comprehensive range of services as an Innovator, Researcher, Embedded Systems Engineer, and Full Stack Developer. From designing PCBs to researching and solving problems, I can develop websites, mobile apps, and cloud-based solutions. With expertise in Docker, Kubernetes, CI/CD, and GIT, I ensure seamless integration and efficient development processes. Trust me to deliver innovative and reliable solutions that meet your specific needs.

Fluent Languages:
Python, Php, JavaScript, C#, C++, Rust, Laravel, Flutter, Flask, CodeIgniter,Next Js, React Js, Nuxt Js, Go.

PCB Electronic Design
Altium Designer, KiCad.

3D Computer Graphics:

Blender, Solid Works.

Phone: +254 706 852 923 / 7 50 515 222,

***@***.*** ***@***.***

"Design is not just what it looks like and feels like. Design is how it works." ―Steve Jobs

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by zaynaab** [comment]
Thank you.
I'm waiting for the update.

**Comment by zaynaab** [comment]
any update?

**Comment by brayton-anderson** [comment]
Hi Zaynab,

Is there a way i can have you on a call to explain my solution.

Kind regards,
Brayton Anderson.

Innovator, Embedded Systems Engineer & Full Stack Developer
I offer a comprehensive range of services as an Innovator, Researcher, Embedded Systems Engineer, and Full Stack Developer. From designing PCBs to researching and solving problems, I can develop websites, mobile apps, and cloud-based solutions. With expertise in Docker, Kubernetes, CI/CD, and GIT, I ensure seamless integration and efficient development processes. Trust me to deliver innovative and reliable solutions that meet your specific needs.

Fluent Languages:
Python, Php, JavaScript, C#, C++, Rust, Laravel, Flutter, Flask, CodeIgniter,Next Js, React Js, Nuxt Js, Go.

PCB Electronic Design
Altium Designer, KiCad.

3D Computer Graphics:

Blender, Solid Works.

Phone: +254 706 852 923 / 7 50 515 222,

***@***.*** ***@***.***

"Design is not just what it looks like and feels like. Design is how it works." ―Steve Jobs

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by zaynaab** [comment]
yes you can tell me your availability we can arrange  zoom call.

**Comment by zaynaab** [comment]
But i would be more helpful if you can update read me file as it will be
useful for me to work in future

**Comment by brayton-anderson** [comment]
The app doesn't host pre-eclampsia data, i wanted to explain how our solution works. How we intend to fetch the data.

Kind regards,
Brayton Anderson.

Innovator, Embedded Systems Engineer & Full Stack Developer
I offer a comprehensive range of services as an Innovator, Researcher, Embedded Systems Engineer, and Full Stack Developer. From designing PCBs to researching and solving problems, I can develop websites, mobile apps, and cloud-based solutions. With expertise in Docker, Kubernetes, CI/CD, and GIT, I ensure seamless integration and efficient development processes. Trust me to deliver innovative and reliable solutions that meet your specific needs.

Fluent Languages:
Python, Php, JavaScript, C#, C++, Rust, Laravel, Flutter, Flask, CodeIgniter,Next Js, React Js, Nuxt Js, Go.

PCB Electronic Design
Altium Designer, KiCad.

3D Computer Graphics:

Blender, Solid Works.

Phone: +254 706 852 923 / 7 50 515 222,

***@***.*** ***@***.***

"Design is not just what it looks like and feels like. Design is how it works." ―Steve Jobs

Sent with [Proton Mail](https://proton.me/) secure email.
